FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -245,7 +245,7 @@ + (instancetype)providerWithAuth:(FIRAuth *)auth {
     @param error The error that occurred if any.
     @return The reCAPTCHA token if successful.
  */
-- (NSString *)reCAPTCHATokenForURL:(NSURL *)URL error:(NSError **)error {
+- (nullable NSString *)reCAPTCHATokenForURL:(NSURL *)URL error:(NSError **_Nonnull)error {
```

Comment:
Same here, it will be safer to check for NULL because there might be cases when compiler and analyzer may not catch nullability issue.

Reply:
The error pointer for the one call is allocated at https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m#L667

Reply:
LGTM, thank you for checking. I sill think it is safer to assume nullable pointer to make it less probable to accidentally introduce a crash in the future.

Reply:
Done in #8214